### DISCUSSION TITLE
Add Timestamp Field

### DISCUSSION THREAD

**Discussion by GabeBrPierce** [community/community#20612] [Other Feature Feedback, Questions, & Ideas]
I Use GitHub projects to collaborate on projects with my clients. It shows them what I am currently working on and what I plan on doing next etc.

I think it would be a great feature to add a type of field that would record the timestamp of when a project was moved from one status to another.
![image](https://user-images.githubusercontent.com/82479780/179262938-4927b4de-9a5a-4cc9-9eac-25e9ef47e5c9.png)

**Comment by doguedogue** [comment]
well, in the history of commits you have this, all movements are recorded in there. 

https://github.com/XXXXX/YYYY/commits/main
